$NetBSD: patch-tools_wscript,v 1.4 2023/01/07 21:20:19 triaxx Exp $

- Add NetBSD support.
- Ensure jack_netsource is built on all Unixes.

--- tools/wscript.orig	2022-04-15 19:14:06.000000000 +0000
+++ tools/wscript
@@ -35,6 +35,8 @@ def build(bld):
         os_incdir = ['../freebsd', '../posix']
     if bld.env['IS_SUN']:
         os_incdir = ['../solaris', '../posix']
+    if bld.env['IS_NETBSD']:
+        os_incdir = ['../netbsd', '../posix']
     if bld.env['IS_WINDOWS']:
         os_incdir = ['../windows']
     for example_tool, example_tool_source in list(example_tools.items()):
@@ -79,6 +81,9 @@ def build(bld):
         prog.use = ['CELT', 'SAMPLERATE', 'OPUS', 'M', 'clientlib']
         prog.target = 'jack_netsource'
         prog.defines = ['HAVE_CONFIG_H']
+        if bld.env['IS_SUN']:
+            prog.env.append_value('LINKFLAGS', '-lnsl')
+            prog.env.append_value('LINKFLAGS', '-lsocket')
 
     if bld.env['IS_LINUX'] and bld.env['BUILD_TOOL_ALSA_IO']:
         prog = bld(features='c cprogram')
